pysma/const.py

```diff
@@ -356,22 +697,94 @@
+        SENSOR_BATTERY_AC_POWER_CHARGE_B,
+        SENSOR_BATTERY_AC_POWER_CHARGE_C,
+        SENSOR_BATTERY_AC_POWER_DISCHARGE_TOTAL,
+        SENSOR_BATTERY_AC_POWER_DISCHARGE_A,
+        SENSOR_BATTERY_AC_POWER_DISCHARGE_B,
+        SENSOR_BATTERY_AC_POWER_DISCHARGE_C,
+        SENSOR_BATTERY_AC_REACTIVE_POWER,
+        SENSOR_BATTERY_AC_REACTIVE_POWER_L1,
+        SENSOR_BATTERY_AC_REACTIVE_POWER_L2,
+        SENSOR_BATTERY_AC_REACTIVE_POWER_L3,
+        SENSOR_BATTERY_AC_APPARENT_POWER,
+        SENSOR_BATTERY_AC_APPARENT_POWER_L1,
+        SENSOR_BATTERY_AC_APPARENT_POWER_L2,
+        SENSOR_BATTERY_AC_APPARENT_POWER_L3,
+        SENSOR_BATTERY_AC_POWER_FACTOR,
+        SENSOR_BATTERY_AC_POWER_FACTOR_EXCITATION,
+        SENSOR_BATTERY_AC_ENERGY_CHARGE_TOTAL,
+        SENSOR_BATTERY_AC_ENERGY_CHARGE_A,
+        SENSOR_BATTERY_AC_ENERGY_CHARGE_B,
+        SENSOR_BATTERY_AC_ENERGY_CHARGE_C,
+        SENSOR_BATTERY_AC_ENERGY_DISCHARGE_TOTAL,
+        SENSOR_BATTERY_AC_ENERGY_DISCHARGE_A,
+        SENSOR_BATTERY_AC_ENERGY_DISCHARGE_B,
+        SENSOR_BATTERY_AC_ENERGY_DISCHARGE_C,
+        SENSOR_BATTERY_AC_ENERGY_DISCHARGE_DAY,
+        SENSOR_BATTERY_SOC_TOTAL,
+        SENSOR_BATTERY_SOC_A,
+        SENSOR_BATTERY_SOC_B,
+        SENSOR_BATTERY_SOC_C,
+        SENSOR_BATTERY_CAPACITY_TOTAL,
+        SENSOR_BATTERY_CAPACITY_A,
+        SENSOR_BATTERY_CAPACITY_B,
+        SENSOR_BATTERY_CAPACITY_C,
         SENSOR_BATTERY_TEMP_A,
         SENSOR_BATTERY_TEMP_B,
         SENSOR_BATTERY_TEMP_C,
-        SENSOR_BATTERY_VOLTAGE_A,
-        SENSOR_BATTERY_VOLTAGE_B,
-        SENSOR_BATTERY_VOLTAGE_C,
-        SENSOR_BATTERY_CURRENT_A,
-        SENSOR_BATTERY_CURRENT_B,
-        SENSOR_BATTERY_CURRENT_C,
+        SENSOR_BATTERY_INSULATION_RESISTANCE,
+        SENSOR_BATTERY_INSULATION_RESIDUAL_CURRENT,
+        SENSOR_BATTERY_OPERATING_TIME_TOTAL,
+        SENSOR_BATTERY_OPERATING_TIME_FEEDIN,
+        SENSOR_BATTERY_STATS_GRID_CONNECTIONS,
+        SENSOR_BATTERY_CONDITION_INVERTER,
+        SENSOR_BATTERY_CONDITION_A,
+        SENSOR_BATTERY_CONDITION_B,
+        SENSOR_BATTERY_CONDITION_C,
+        SENSOR_BATTERY_CONDITION_TEMP_A,
+        SENSOR_BATTERY_CONDITION_TEMP_B,
+        SENSOR_BATTERY_CONDITION_TEMP_C,
+        SENSOR_BATTERY_STATUS_GENERAL,
+        SENSOR_BATTERY_STATUS_BMS_A,
+        SENSOR_BATTERY_STATUS_BMS_B,
+        SENSOR_BATTERY_STATUS_BMS_C,
+        SENSOR_BATTERY_STATUS_OPERATING_MODE,
+        SENSOR_BATTERY_STATUS_GRID_SWITCH,
+        SENSOR_BATTERY_STATUS_POWER_SUPPLY,
+        SENSOR_BATTERY_STATUS_FIRMWARE_UPDATE,
+        SENSOR_BATTERY_STATUS_SYSTEM_INIT,
+        SENSOR_BATTERY_STATUS_CHARGE_CONTROL_COMM,
         SENSOR_GRID_POWER,
+        SENSOR_VOLTAGE_L1,
+        SENSOR_VOLTAGE_L2,
+        SENSOR_VOLTAGE_L3,
+        SENSOR_CURRENT_L1,
+        SENSOR_CURRENT_L2,
+        SENSOR_CURRENT_L3,
+        SENSOR_CURRENT_TOTAL,
+        SENSOR_FREQUENCY,
         SENSOR_STATUS,
     ],
```

Comment:
This is quite a large list of sensors. Are you sure you need all of them? As all sensors from this list will be created in Home Assistant.